### PR TITLE
Feature/style profile/179060260

### PR DESCRIPTION
**Pull request by claireclark1**
Purpose
---------------
[https://www.pivotaltracker.com/story/show/179060260](url)
feature

Solution
---------------
- follow wire frames for profile styling 
=> [https://app.zeplin.io/project/60a3f1172759472a903908f7/screen/61019a2e8fe459a141e4a51d]()

Change summary
---------------
* styled profile
* moved components around a little to make styling easier
* changed tests to get them passing

Steps to Verify
----------------
1. Check that it works in your browser
2. read code

Additional details / screenshot
----------------
** design was slightly altered because there were issues that would have came about in edge cases that would have broken the layout in the wire frame.
https://user-images.githubusercontent.com/43625033/128414154-3f307d26-f3d1-4cc2-b679-7a1aadd91079.mov

### CODE REVIEW

File: src/test/testFeeds.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const generatePerson = (name?: string): ActivityContentProfile => {
 };
 
 /**
- * Generate a FeedItem needed to make a full Feed
+ * Generate a FeedItem needed to make a full FeedNavigation
```
Comment (shannonwells) [inline src/test/testFeeds.ts]: I think this is from a Replace All and doesn't need to change.

File: src/utilities/types.d.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+import {
+  ActivityContentNote,
+  ActivityContentProfile,
+} from "@dsnp/sdk/core/activityContent";
 
 export type HexString = string;
 export type EncryptedString = string;
 export type URLString = string;
-// ### Feed Data Types ###
+// ### FeedNavigation Data Types ###
```
Comment (shannonwells) [inline src/utilities/types.d.ts]: Same here, I'm pretty sure these two should just be "Feed."

File: src/components/PostList.tsx [hunk 3]
```diff
@@ -72,7 +72,7 @@ const PostList = ({ feedType }: PostListProps): JSX.Element => {
             })}
         </>
       ) : (
-        "Empty Feed!"
+        "Empty FeedNavigation!"
```
Comment (shannonwells) [inline src/components/PostList.tsx]: Shouldn't this just stay as  "Empty Feed"?

**Review by shannonwells — CHANGES_REQUESTED** [review]
There are a few places where Feed should not replaced by FeedNavigation.  Other than that the only feedback I have is non-blocking, which is it would be nice to have the entire area that you click to slide the profile out, be clickable. When I moved the cursor over it it didn't turn to a pointer and it was a bit confusing. I think the whole square should be clickable (I tried to highlight it):

Screen Shot 2021-08-05 at 3 20 14 PM

File: src/components/PostList.tsx (hunk 3 above)
Comment (claireclark1) [reply src/components/PostList.tsx]: wow webstorm is ultra good at refactoring

**Review by shannonwells — APPROVED** [review]
🏅
